timesketch/frontend-v3/src/layouts/AppBar.vue

```diff
@@ -29,17 +29,62 @@ limitations under the License.
       >
       </v-img>
     </template>
-    <template v-slot:append> Right hand side </template>
+    <template v-slot:append>
+      <v-avatar color="grey lighten-1" size="25" class="ml-3">
+        <span class="white--text">{{ $filters.initialLetter(currentUser) }}</span>
+      </v-avatar>
+      <v-menu offset-y>
+        <template v-slot:activator="{ props }">
+          <v-btn small icon v-bind="props">
+            <v-icon title="Timesketch Options">mdi-dots-vertical</v-icon>
+          </v-btn>
+        </template>
+          <v-list>
+            <v-list-item
+              prepend-icon="mdi-brightness-6"
+              v-on:click="toggleTheme"
+            >
+              <v-list-item-title>Toggle theme</v-list-item-title>
+            </v-list-item>
+            <v-list-item
+              href="/legacy/"
+              prepend-icon="mdi-view-dashboard-outline"
+            >
+              <v-list-item-title>Use the old UI</v-list-item-title>
```

Comment:
I wonder if we should skip this for the new UI? @jkppr WDYT?

Reply:
Yes, I think with v3 we can finally deprecate the old legacy front-end.

Do we need a link to the frontend-ng folder though until we have everything ported?

Reply:
The route does not point anywhere yet, so we can still decide later what we mean by "old ui". I'd say we definitely don't need v1, but maybe ng is a good idea for the time of migration.

Reply:
Sounds good